--- origsrc/qtoctave-0.8.2/easy_plot/src/CMakeLists.txt	2009-03-27 04:46:45.000000000 -0500
+++ src/qtoctave-0.8.2/easy_plot/src/CMakeLists.txt	2009-09-24 15:28:59.354557500 -0500
@@ -13,6 +13,7 @@ include_directories(${QT_QTCORE_INCLUDE_
 include_directories(${QT_QTGUI_INCLUDE_DIR})
 include_directories(${QT_QTXML_INCLUDE_DIR})
 include_directories(${QT_QTSVG_INCLUDE_DIR})
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
 QT4_WRAP_UI(ui_design_main_window.h design_main_window.ui)
 QT4_WRAP_UI(ui_export_to_dialog.h export_to_dialog.ui)
@@ -42,6 +43,6 @@ add_executable(easy_plot 
 TARGET_LINK_LIBRARIES(easy_plot ${Qt4_LIBRARIES} ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY} ${QT_QTSQL_LIBRARY} ${QT_QTSVG_LIBRARY})
 
 install	(
-		PROGRAMS easy_plot
+		TARGETS easy_plot
 		DESTINATION bin
-	)
\ No newline at end of file
+	)
--- origsrc/qtoctave-0.8.2/qtjs/src/CMakeLists.txt	2009-06-28 16:53:40.000000000 -0500
+++ src/qtoctave-0.8.2/qtjs/src/CMakeLists.txt	2009-09-24 15:23:08.310443700 -0500
@@ -37,13 +37,12 @@ SET_TARGET_PROPERTIES(qtjs
 )
 
 TARGET_LINK_LIBRARIES(qtjs 
-	${Qt4_LIBRARIES} 
+	${QT_QTUITOOLS_LIBRARY}
 	${QT_QTCORE_LIBRARY} 
 	${QT_QTGUI_LIBRARY} 
 	${QT_QTXML_LIBRARY} 
 	#${QT_QTSVG_LIBRARY} 
 	${QT_QTSCRIPT_LIBRARY}
-	${QT_QTUITOOLS_LIBRARY}
 )
 
 
--- origsrc/qtoctave-0.8.2/qtoctave/src/operations/general_menu.h	2009-05-28 10:16:12.000000000 -0500
+++ src/qtoctave-0.8.2/qtoctave/src/operations/general_menu.h	2009-09-24 15:26:36.934881400 -0500
@@ -1,4 +1,4 @@
-#include "../ui_general_menu.h"
+#include "ui_general_menu.h"
 #include "../config.h"
 #include <QFont>
 
--- origsrc/qtoctave-0.8.2/qtoctave/src/projects/projects.cpp	2009-05-28 10:16:12.000000000 -0500
+++ src/qtoctave-0.8.2/qtoctave/src/projects/projects.cpp	2009-09-24 15:26:55.727668600 -0500
@@ -1,5 +1,5 @@
 #include "projects.h"
-#include "../ui_newprojectdialog.h"
+#include "ui_newprojectdialog.h"
 #include <QXmlStreamReader>
 #include <QXmlStreamWriter>
 #include <QDir>
--- origsrc/qtoctave-0.8.2/qtoctave/src/projects/projects.h	2009-05-28 10:16:12.000000000 -0500
+++ src/qtoctave-0.8.2/qtoctave/src/projects/projects.h	2009-09-24 15:25:11.867331600 -0500
@@ -19,7 +19,7 @@
 #ifndef __PROJECTS_H__
 #define __PROJECTS_H__
 
-#include "../ui_projects.h"
+#include "ui_projects.h"
 #include "../config.h"
 
 using namespace Ui;
--- origsrc/qtoctave-0.8.2/qtoctave/src/svgcanvas/svgcanvas.cpp	2009-05-28 10:16:12.000000000 -0500
+++ src/qtoctave-0.8.2/qtoctave/src/svgcanvas/svgcanvas.cpp	2009-09-24 15:27:58.697380700 -0500
@@ -235,7 +235,7 @@ void SvgCanvas::export_to_pdf_ps(int wid
 	plot.end();
 }
 
-#include "../ui_export_to_dialog.h"
+#include "ui_export_to_dialog.h"
 
 enum ExportFormat {PIXMAP, PICTURE, PRINTER, PDF, SVG};
 
